pkg/codegen/schema.go

```diff
@@ -52,6 +62,52 @@ func (s Schema) GetAdditionalTypeDefs() []TypeDefinition {
 	return result
 }
 
+func (s Schema) EnumValues() []Enum {
+	if s.GoType != "string" && !strings.HasPrefix(s.GoType, "int") {
```

Comment:
We should allow enums for all types. IMO. No sense limiting to strings and ints.